data/template/14882.css

```diff
@@ -259,15 +259,13 @@ code.itemdeclcode {
 	display: block;
 }
 
-span.comment { color: green; font-style: italic; font-family: serif; }
-span.keyword { color: #00607c; }
-span.parenthesis { color: #af1915; }
-span.curlybracket { color: #af1915; }
-span.squarebracket { color: #af1915; }
-span.literal { color: #9F6807; }
-span.operator { color: #570057; }
-span.anglebracket { color: #570057; }
-span.preprocessordirective { color: #6F4E37; }
```

Comment:
pandoc's codegen uses the abbreviated names on the right, not the friendly names on the left. Probably the real way to do this is to change the `highlight-style`, which is explicitly set in `filter/wg21.py` right now and does stuff like:

```py
text_styles['ControlFlow'] = text_styles['DataType']
text_styles['Keyword'] = text_styles['DataType']
```

maybe that should just be:


```py
text_styles['ControlFlow'] = '#00607c'
text_styles['Keyword'] = '#00607c'
```

But I don't know what that'll do to your pdf.

Reply:
I've now pushed a `data/syntax/wg21.theme` file which captures this. We could/should remove this CSS in PR though so that we aren't confused.